.github/workflows/build.yaml

```diff
@@ -0,0 +1,34 @@
+name: build-refdocs
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+  pull_request:
+
+jobs:
+  build:
+    name: Build reference docs
+    strategy:
+      matrix:
+        go-version: [1.15.x]
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+    env:
+      KUBE_VERSION: 1.20
+      BUILD_PATH: /tmp
+    steps:
+      - name: Set up Go ${{ matrix.go-version }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+        id: go
+      - name: Build
+        run: |
+          mkdir -p ${BUILD_PATH}/src/k8s.io
```

Comment:
Do we have BUILD_PATH set somewhere?
In travis, we have global variables. Is it possible to do that in this workflow?

Reply:
I used /tmp. I am not sure what to do here.
I would like to run a script to read a configuration file containing say, `version`?

Reply:
Could add a config.toml containing BUILD_PATH and RELEASE_VERSION?

Reply:
It would be good to have configuration separated from the code (yaml workflow), although I'm not sure how to do that with workflows.

Reply:
Yes. Nice to read in a property file.